src/main/webapp/app/entities/course/course.component.html

```diff
@@ -45,7 +45,7 @@ <h4 id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses
                         ><fa-icon icon="sort"></fa-icon>
                     </th>
                     <th jhiSortBy="presentationScoreEnabled">
-                        <span>{{ 'artemisApp.course.presentationScoreEnabled' | translate }}</span
+                        <span>{{ 'artemisApp.course.presentationScoreEnabled.title' | translate }}</span
```

Comment:
I noticed during testing this bug fix that the translation key is wrong and solved it in this PR as well.